From 28b1fa54f0c606281033ad56046af410756171e6 Mon Sep 17 00:00:00 2001
From: Caolan McNamara <caolanm@redhat.com>
Date: Sat, 3 Jan 2009 19:12:18 -0600
Subject: [PATCH 09/15] fix the AST_PROG_SED problem that makes ./bootstrap.sh fail

Need to regenerate configure from configure.ac after patch one with
./bootsrap.sh, so fix the broken .m4 in the autoconfig dir. (alternatively
change AST_PROG_SED to AC_PROG_SED as it is the same thing in the other .m4 in
the autoconf dir)
---
 autoconf/ast_prog_sed.m4 |    2 +-
 1 files changed, 1 insertions(+), 1 deletions(-)

diff --git a/autoconf/ast_prog_sed.m4 b/autoconf/ast_prog_sed.m4
index 283d52f..ae6cfc0 100644
--- a/autoconf/ast_prog_sed.m4
+++ b/autoconf/ast_prog_sed.m4
@@ -12,7 +12,7 @@ AC_DEFUN([AST_PROG_SED],
      done
      echo "$ac_script" | sed 99q >conftest.sed
      $as_unset ac_script || ac_script=
-     _AC_PATH_PROG_FEATURE_CHECK(SED, [sed gsed],
+     _AC_PATH_PROGS_FEATURE_CHECK(SED, [sed gsed],
 	[_AC_FEATURE_CHECK_LENGTH([ac_path_SED], [ac_cv_path_SED],
 		["$ac_path_SED" -f conftest.sed])])])
  SED="$ac_cv_path_SED"
-- 
1.6.1

